src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -276,7 +279,7 @@ public enum HolonPhantoms implements LogicCardInfo {
           actionA {
             assert bg.em().retrieveObject("Form_Change") != bg.turnCount : "You can’t use more than 1 Form Change Poké-Power each turn"
             checkLastTurn()
-            assert my.deck : "Deck is empty"
+            assert my.deck : "Deck is empty!"
```

Comment:
The assert wcu still adds a . at the end doesn't it? "Deck is empty!." will probably look awkward if so.

Reply:
Yes that is correct, I've reverted this